README.md

```diff
@@ -41,12 +41,18 @@ To submit a new RFC, follow these steps:
 1. Copy the appropriate template file from the `templates` directory into the appropriate `designs` subdirectory (such as `designs/2018-typescript-support/README.md`). Be sure to name your file `README.md` so it is easily viewable in the GitHub interface.
 1. If you want to include images in your RFC, place them in the same directory as the `README.md`.
 1. Fill in the RFC. Please fill in every section in the template with as much detail as possible.
-1. Submit a pull request to this repo with all of your files.
-1. You will receive feedback both from the ESLint community and from the ESLint team. You should be prepared to update your RFC based on this feedback. The goal is to build consensus on the best way to implement the suggested change.
-1. When all feedback has been incorporated, the ESLint TSC will determine whether or not to accept the RFC.
+1. Submit a pull request to this repo with all of your files. This begins the approval process (detailed below).
 1. RFCs that are accepted will be merged directly into this repo; RFCs that are not accepted will have their pull requests closed without merging.
 
-## The RFC life-cycle
+## The RFC Approval Process
+
+When an RFC is submitted, it goes throw the following process:
+
+1. **Initial commenting period (21 days)** - the community and ESLint team are invited to provide feedback on the proposal. During this period, you should expect to update your RFC based on the feedback provided. Very few RFCs are ready for approval without edits, so this period is important for fine-tuning ideas and build consensus. (A PR in the initial commenting period has the **Initial Commenting** label applied.) The initial commenting period may be extended with a TSC vote if the RFC has been unable to reach consensus but the team believes it worthwhile to continue developing the RFC. The TSC may also decide to abandon an RFC that has failed to reach consensus after 21 days.
```

Comment:
I'm not sure I understand this section. If an RFC has been open for 21 days, does this mean the TSC is required to take some action on it (either voting to extend the commenting period or abandoning it)? Would the RFC automatically be added to the TSC agenda so that the TSC can decide which action to take?

Given the time required to create an RFC and update it in response to suggested changes, my impression is that 21 days might be too short a timespan for the development of most RFCs (e.g. https://github.com/eslint/rfcs/pull/4, https://github.com/eslint/rfcs/pull/5, and https://github.com/eslint/rfcs/pull/6 have already been open for over a month).

Reply:
Yeah, that's something I wanted to get more opinions on. My thought was that the initial commenting period should be long enough that we can't be accused of rushing the RFC through. For instance, I don't think we'd want to push an RFC through to final commenting after just 7 days because if people are on vacation or there are holidays, not everyone would have time to add feedback.

So maybe we should set a minimum numbers of days for the initial commenting period and allow it to last as long as feedback comes in?

Reply:
> So maybe we should set a minimum numbers of days for the initial commenting period and allow it to last as long as feedback comes in?

Sounds good to me.